packages/core/src/api/channel.ts

```diff
@@ -708,4 +709,43 @@ export class ChannelsAPI {
 			signal,
 		}) as Promise<RESTPostAPISendSoundboardSoundResult>;
 	}
+
+	/**
+	 * Adds a recipient to a group DM channel
+	 *
+	 * @param channelId - The id of the channel to add the recipient to
+	 * @param userId - The id of the user to add as a recipient
+	 * @param body - The data for adding the recipient
+	 * @param options - The options for adding the recipient
+	 */
+	public async addGroupDMRecipient(
+		channelId: Snowflake,
+		userId: Snowflake,
+		body: RESTPutAPIChannelRecipientJSONBody,
+		{ auth, signal }: Pick<RequestData, 'auth' | 'signal'> = {},
+	) {
+		await this.rest.put(Routes.channelRecipient(channelId, userId), {
+			auth,
+			body,
+			signal,
+		});
+	}
+
+	/**
+	 * Removes a recipient from a group DM channel
+	 *
+	 * @param channelId - The id of the channel to remove the recipient from
+	 * @param userId - The id of the user to remove as a recipient
+	 * @param options - The options for removing the recipient
+	 */
+	public async deleteGroupDMRecipient(
```

Comment:
Would "remove" here be more consistent?

https://github.com/discordjs/discord.js/blob/ffbb7b69366b61ff9433adaef7de28db327315ba/packages/core/src/api/guild.ts#L1224-L1238